templates/default/slurm.conf.erb

```diff
@@ -66,7 +66,8 @@ SchedulerType=sched/backfill
 #SchedulerAuth=
 #SchedulerPort=
 #SchedulerRootFilter=
-SelectType=select/cons_res
+GresTypes=gpu
```

Comment:
Let's start organizing the config into sections, can we group the GPU related options under something like 
```
### TRES and GPU config options
```